sdk/storage/Azure.Storage.Blobs/src/Generated/BlobRestClient.cs

```diff
@@ -55,10 +57,10 @@ public static partial class Service
                     {
                         Azure.Response _response = async ?
                             // Send the request asynchronously if we're being called via an async path
-                            await pipeline.SendRequestAsync(_request, cancellationToken).ConfigureAwait(false) :
+                            await pipeline.SendRequestAsync(_request, bufferResponse, cancellationToken).ConfigureAwait(false) :
                             // Send the request synchronously through the API that blocks if we're being called via a sync path
                             // (this is safe because the Task will complete before the user can call Wait)
-                            pipeline.SendRequest(_request, cancellationToken);
+                            pipeline.SendRequest(_request, bufferResponse, cancellationToken);
```

Comment:
Nit - parameters should each have a new line

Reply:
Same

Reply:
Oh right my bad.